### PR TITLE
Definition of Transparent Log Signature Structure

### PR DESCRIPTION
**Pull request by fishseabowl**
## Description

This PR just defines the transparent log signature structure

## PR Checklist

- [x] I've read the [contributing guidelines](https://github.com/pyrsia/.github/blob/main/contributing.md).
- [x] I've read ["What is a Good PR?"](https://github.com/pyrsia/pyrsia/blob/main/docs/get_involved/good_pr.md)
- [x] I've included a good title and brief description along with how to review them.
- [x] I've linked any associated an [issue](https://github.com/pyrsia/pyrsia/issues).

### Code Contributions

- [x] I've built the code `cargo build --all-targets` successfully.
- [ ] I've run the unit tests `cargo test --workspace` and everything passes.
- [x] I've made sure my rust toolchain is current `rustup update`.

### CODE REVIEW

File: src/transparency_log/log.rs [hunk 1]
```diff
@@ -29,19 +29,31 @@ enum TransparencyLogError {
...
     operation: Operation,
 }
 
+#[derive(Serialize, Deserialize, Debug, Clone)]
+pub struct SignatureEnvelope {
+    /// The data that is integrity protected
+    payload: Payload,
+    /// The time at which the signature was generated. This is a part of signed attributes
+    signing_timestamp: u64,
+    /// The digital signatres computed on payload and signed attributes
```
Comment (johanvos) [inline src/transparency_log/log.rs:51]: signature

**Review by johanvos — APPROVED** [review]
I think this is a good start. I have no idea if that structure is the ultimate one that we should use in the end, but I also have no reason why this is not correct, so it looks good.
If it turns out we need more info in that envelope, or that we want to change part of the structure, we can always revisit this.

File: src/transparency_log/log.rs (hunk 1 above)
Comment (fishseabowl) [reply src/transparency_log/log.rs]: fixed. thanks

**Review by mseabornIBM — DISMISSED** [review]

**Review by sbtaylor15 — APPROVED** [review]
/lgtm

**Review by sbtaylor15 — APPROVED** [review]
/lgtm

**Review by tiainen — APPROVED** [review]

**Pull request merged**